src/clj/rems/api/schema.clj

```diff
@@ -16,9 +16,33 @@
    :localizations (s/maybe {s/Any s/Any})})
 
 (def License
+  {:id s/Num
+   :licensetype (s/enum "text" "link" "attachment")
+   :title s/Str
+   :textcontent s/Str
+   :localizations {s/Keyword {:title s/Str :textcontent s/Str}}})
```

Comment:
It's good to know that license also has start & end even though they are not used

Reply:
done